### ISSUE TITLE
Email Board + Leader for Team Changes

### ISSUE THREAD

**Issue by Jambrose777** [thewca/worldcubeassociation.org#4430]
Whenever a change happens on the edit teams page, the Board should be notified of the change.

CCed in email: Board, Leader of team (or potential other party, like WDC for banned), and the changer.

**Comment by jfly** [comment]
This is moderately tricky because of how we store historical changes. It's not obvious from the API call the user makes *what* exactly changed.

**Comment by AlbertoPdRF** [comment]
Yes; one possibility that was discussed in the past was the creation of a weekly or monthly newsletter that would mention any members whose start and/or end dates fall within the corresponding period of time. There may be other options though!

**Comment by jfly** [comment]
Oh, I like that idea! It also behaves nicely if someone makes some mistakes and then fixes them immediately: we won't unnecessarily spam people.

Another idea entirely: we've talked about redesigning the team page to distinguish between "normal administrative tasks" and "changing history to fix mistakes/add missing data". If we do that, then we'd end up creating APIs where it would be more possible to implement what @Jambrose777 described originally.

**Comment by AlbertoPdRF** [comment]
I like your idea too! :stuck_out_tongue:

**Comment by Jambrose777** [comment]
I'd recommend doing #6083 before ever tackling this, as we probably wouldn't want to flood email boxes with historical changes.

**Comment by maxidragon** [comment]
This issue can be closed, as we now have role changes emails
